### PR TITLE
Add missing undiscounted total price in order line

### PR DESCRIPTION
**Pull request by kadewu**
Fixes: https://github.com/saleor/saleor/issues/14030

The resolver was already implemented, the field was just missing from our schema.

# Impact

- [ ] New migrations
- [ ] New/Updated API fields or mutations
- [ ] Deprecated API fields or mutations
- [ ] Removed API types, fields, or mutations

# Docs

- [ ] Link to documentation:

# Pull Request Checklist

- [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
- [ ] Database queries are optimized and the number of queries is constant
- [ ] Database migrations are either absent or optimized for zero downtime
- [ ] The changes are covered by test cases

### CODE REVIEW

File: saleor/graphql/order/types.py [hunk 1]
```diff
@@ -654,6 +654,13 @@ class OrderLine(ModelObjectType[models.OrderLine]):
     total_price = graphene.Field(
         TaxedMoney, description="Price of the order line.", required=True
     )
+    undiscounted_total_price = graphene.Field(
+        TaxedMoney,
+        description=(
+            "Price of the order line without applied and order line discount."
```
Comment (zedzior) [inline saleor/graphql/order/types.py:660]: Something is wrong with the sentence.

Comment (kadewu) [reply saleor/graphql/order/types.py]: Right, misspelling. Thanks for catching.

Comment (ritanjandawn) [reply saleor/graphql/order/types.py]: Perhaps an alternative  "Price without applying an order line discount"

**Review by zedzior — APPROVED** [review]

**Review by michal-macioszczyk — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/order/types.py [hunk 2]
```diff
@@ -654,6 +654,11 @@ class OrderLine(ModelObjectType[models.OrderLine]):
     total_price = graphene.Field(
         TaxedMoney, description="Price of the order line.", required=True
     )
+    undiscounted_total_price = graphene.Field(
+        TaxedMoney,
+        description=("Price of the order line without applied an order line discount."),
```
Comment (fowczarek) [inline saleor/graphql/order/types.py:659]: 👮‍♂️ 🚓 
```suggestion
        description="Price of the order line without applied an order line discount.",
```

Comment (fowczarek) [reply saleor/graphql/order/types.py:659]: Also maybe 
`Price of the order line without discounts.`

Comment (kadewu) [reply saleor/graphql/order/types.py:659]: I was trying to stay "close" to the other descriptions, but I'm fine with changing it.

**Review by yellowee — APPROVED** [review]

**Pull request merged**